app/views/application/_footer.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="footer-basic">
+    <footer>
+        <ul>
+            San Francisco Bay Bird Observatory ● <a href="https://maps.google.com/?q=524+Valley+Way,+Milpitas,+CA+95036&amp;entry=gmail&amp;source=g"> 524 Valley Way, Milpitas, CA 95035 </a> ● 408-946-6548 ● <a href="mailto:outreach@sfbbo.org" target="_blank">outreach@sfbbo.org</a>
```

Comment:
This is still invalid HTML, since it's a child of a ul that is not inside an li.
I suggest either:
* Using the same sort of ul/li structure with the "list-line" and "list-inline-item" classes that are used at the bottom of this file
* Not putting this in a UL, and instead just wrapping in a p tag.